tests/foreman/cli/test_fact.py

```diff
@@ -61,3 +63,46 @@ def test_negative_list_by_name(module_target_sat):
     assert (
         module_target_sat.cli.Fact().list(options={'search': f'fact={gen_string("alpha")}'}) == []
     )
+
+
+@pytest.mark.no_containers
```

Comment:
why no_containers?

Reply:
what regression you mean, could you elaborate if you've example? 
with facts I don't think there should be any challenge using containers, also afaik we're using fake hosts as well for facts in few tests

Reply:
I can't recall it atm but adding another interface fails with container here.

Reply:
@Gauravtalreja1 Can we resolve this?